version/build.go

```diff
@@ -1,7 +1,7 @@
 package version
 
 // VERSION is the main CLI version number. It's defined at build time using -ldflags
-var VERSION = "2.6.1"
+var VERSION = "2.7.0"
```

Comment:
cc: @bvatai-br we should bump this further to 2.8.0 as we already have a [2.7.0 release](https://github.com/bitrise-io/bitrise/releases/tag/2.7.0) (that release must have accidentally went out without this change cc: @ofalvai)